cloudinit/config/cc_mounts.py

```diff
@@ -257,6 +257,7 @@ def create_swap(fname, size, method):
         except subp.ProcessExecutionError as e:
             LOG.warning(errmsg, fname, size, method, e)
```

Comment:
you should not log a warning here.
let the caller decide what to do with it.

because in this case, if fallocate fails, it is not warning level. but if dd fails after fallocate, then it is.

only the caller knows.